### PR TITLE
preliminary support for meta feeds

### PR DESCRIPTION
**Pull request by staltz**
- [x] ~~publishAs supports meta feeds~~
- [x] add support meta feeds
- [ ] addOOO supports meta feeds

### CODE REVIEW

File: test/basic.js [hunk 1]
```diff
@@ -235,11 +303,30 @@ test('validate needs to load', (t) => {
...
     })
   })
 })
+
+test('publishAs classic', (t) => {
+  const keys = ssbKeys.generate()
+
+  const content = { type: 'post', text: 'hello world!' }
+
+  db.publishAs(keys, content, (err, msg) => {
```
Comment (arj03) [inline test/basic.js:320]: A better test case here is to add 2 messages and check that sequence on the second is correct. Just found a bug in the other branch related to that problem ;)

**Comment by staltz** [comment]
I'm working on rebasing this branch on master after we added validate2-rsjs.

**Comment by staltz** [comment]
@arj03 Do you greenlight closing this PR and branch?

**Comment by arj03** [comment]
Yes please

**Pull request closed without merging**